pyproject.toml

```diff
@@ -0,0 +1,125 @@
+[build-system]
+requires = ['setuptools', 'wheel']
+build-backend = 'setuptools.build_meta'
+
+[project]
+name = 'ctgan'
+description = 'Create tabular synthetic data using a conditional GAN'
+authors = [{ name = 'DataCebo, Inc.', email = 'info@sdv.dev' }]
+classifiers = [
+    'Development Status :: 2 - Pre-Alpha',
+    'Intended Audience :: Developers',
+    'License :: Free for non-commercial use',
+    'Natural Language :: English',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3.11',
+    'Topic :: Scientific/Engineering :: Artificial Intelligence',
+]
+keywords = ['ctgan', 'CTGAN']
+version = '0.9.1.dev0'
+license = { text = 'BSL-1.1' }
+requires-python = '>=3.8,<3.12'
+readme = 'README.md'
+dependencies = [
+    "numpy>=1.20.0,<2;python_version<'3.10'",
+    "numpy>=1.23.3,<2;python_version>='3.10'",
+    "pandas>=1.1.3;python_version<'3.10'",
+    "pandas>=1.3.4;python_version>='3.10' and python_version<'3.11'",
+    "pandas>=1.5.0;python_version>='3.11'",
+    "scikit-learn>=0.24,<2;python_version<'3.10'",
+    "scikit-learn>=1.1.3,<2;python_version>='3.10'",
+    "torch>=1.8.0;python_version<'3.10'",
+    "torch>=1.11.0;python_version>='3.10' and python_version<'3.11'",
+    "torch>=2.0.0;python_version>='3.11'",
+    'tqdm>=4.15,<5',
+    'rdt>=1.6.1,<2.0',
+]
+
+[project.urls]
+"Source Code"= "https://github.com/sdv-dev/CTGAN/"
+"Issue Tracker" = "https://github.com/sdv-dev/CTGAN/issues"
```

Comment:
```suggestion
"Issue Tracker" = "https://github.com/sdv-dev/CTGAN/issues"
"Changes" = "https://github.com/sdv-dev/CTGAN/blob/main/HISTORY.md"
```

Reply:
yes, done in [ebc9cda](https://github.com/sdv-dev/CTGAN/pull/338/commits/ebc9cda2b8ad63d611fbf1298f3fc29845476d52)